pkgs/tools/games/steamtinkerlaunch/default.nix

```diff
@@ -0,0 +1,105 @@
+{ bash
+, coreutils
+, gamemode
+, gamescope
+, gawk
+, gdb
+, git
+, gnugrep
+, gnused
+, fetchFromGitHub
+, imagemagick
+, installShellFiles
+, jq
+, lib
+, libnotify
+, makeWrapper
+, mangohud
+, nettools
+, p7zip
+, pev
+, procps
+, replay-sorcery
+, rsync
+, scummvm
+, stdenv
+, strace
+, usbutils
+, unixtools
+, unzip
+, vkBasalt
+, wget
+, wine
+, winetricks
+, xdg-utils
+, xdotool
+, xorg
+, yad
+}:
+stdenv.mkDerivation rec {
+  pname = "steamtinkerlaunch";
+  version = "11.11";
+
+  src = fetchFromGitHub {
+    owner = "sonic2kk";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-rWENtgV6spokzkhnmrrzsAQ19dROJ50ofEulU5Jx5IE=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  PREFIX = "\${out}";
+
+  postInstall = ''
+    wrapProgram $out/bin/steamtinkerlaunch \
+      --set SYSTEMSTLCFGDIR $out/share \
+      --prefix PATH : ${lib.makeBinPath [
+      # boxtron
+      # nyrna
+      # vr-video-player
+      bash
+      coreutils
+      gamemode
+      gamescope
+      gawk
+      gdb
+      git
+      gnugrep
+      gnused
+      imagemagick
+      jq
+      libnotify
+      mangohud
+      nettools
+      p7zip
+      pev
+      procps
+      replay-sorcery
+      rsync
+      scummvm
+      strace
+      usbutils
+      unixtools.xxd
+      unzip
+      vkBasalt
+      wget
+      wine
+      winetricks
+      xdg-utils
+      xdotool
+      xorg.xprop
+      xorg.xrandr
+      xorg.xwininfo
+      yad
```

Comment:
Do we really need **all** of this including rsync, strace and gdb?

Reply:
Yes, we should reduce that list to a sensible default. I would suggest to remove all the debugging utilties.

Reply:
what is the canonical way to pin optional dependencies, a second package called `steamtinkerlaunch-full`?